lib/Doctrine/Inflector/Rules/Dutch/Inflectible.php

```diff
@@ -0,0 +1,168 @@
+
+        yield new Transformation(new Pattern('()((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))en$'), '\1\2');
+    }
+
+    /**
+     * @return iterable<Transformation>
+     */
+    public static function getPlural(): iterable
+    {
+        // @todo already in plural (?)
+        // @todo refine
+        yield new Transformation(new Pattern('()(e)(s)$'), '\1\2\3\3en');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Klinkerverandering
+        yield new Transformation(new Pattern('()heid$'), '\1heden');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Beroepen_eindigend_op_-man
+        yield new Transformation(new Pattern('()man$'), '\1mannen');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Latijnse_meervoudsvormen
+        yield new Transformation(new Pattern('()ix$'), '\1ices');
+
+        yield new Transformation(new Pattern('()ex$'), '\1ices');
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Stapelmeervoud
+        yield new Transformation(new Pattern('^(ei|gemoed|goed|hoen|kind|lied|rad|rund)$'), '\1eren');
+
+        // http://nl.wikipedia.org/wiki/Nederlandse_grammatica
+        yield new Transformation(new Pattern('()ij$'), '\1ijen');
+
+        yield new Transformation(new Pattern('()orie$'), '\1orieen');   // ën klemtoon
+
+        yield new Transformation(new Pattern('()io$'), '\1io\'s');
+
+        yield new Transformation(new Pattern('()(a|e|i|o|u|ij)$'), '\1\2s');
+
+        yield new Transformation(new Pattern('()(((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))e((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g)))$'), '\1\2s');
+
+        yield new Transformation(new Pattern('()(((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))(u|i|e|a|o)s)$'), '\1\2sen');
+
+        yield new Transformation(new Pattern('()(((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))s)$'), '\1\2en');
+
+        yield new Transformation(new Pattern('()s$'), '\1zen');
+
+        yield new Transformation(new Pattern('()((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))$'), '\1\2en');
+    }
+
+    /**
+     * @return iterable<Substitution>
+     */
+    public static function getIrregular(): iterable
+    {
+        // http://nl.wikipedia.org/wiki/Klemtoon
+        yield new Substitution(new Word('olie'), new Word('oliën'));
+
+        yield new Substitution(new Word('industrie'), new Word('industrieën'));
+
+        yield new Substitution(new Word('idee'), new Word('ideeën'));
+
+        // @todo: above 3 examples maybe could be compacted into a rule
+
+        // http://nl.wikipedia.org/wiki/Meervoud_(Nederlands)#Klinkerverandering
+        yield new Substitution(new Word('lid'), new Word('leden'));
+
+        yield new Substitution(new Word('smid'), new Word('smeden'));
```

Comment:
This and the word above should be a rule. Also includes words like `ooglid` (and other samples of `~lid`)